dockerfiles/Dockerfile.dev

```diff
@@ -3,8 +3,11 @@ FROM    golang:1.8.3-alpine
 
 RUN     apk add -U git make bash coreutils
 
-RUN     go get github.com/LK4D4/vndr && \
-        cp /go/bin/vndr /usr/bin && \
+ARG     VNDR_COMMIT=9909bb2b8a0b7ea464527b376dc50389c90df587
```

Comment:
@mlaventure this is the docker/cli repo, there's no `hack/dockerfile/binaries-commits`, so I just put it in the dockerfile